<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGroupDialog</class>
 <widget class="QDialog" name="AddGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add person</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelID">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineID">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelColor">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="layoutColor">
       <item>
        <widget class="QLineEdit" name="lineColor">
         <property name="text">
          <string>#00ff00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonColor">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineID</tabstop>
  <tabstop>lineName</tabstop>
  <tabstop>lineColor</tabstop>
  <tabstop>buttonColor</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddGroupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddGroupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineColor</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddGroupDialog</receiver>
   <slot>updateColorLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonColor</sender>
   <signal>clicked()</signal>
   <receiver>AddGroupDialog</receiver>
   <slot>openColorWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateColorLine()</slot>
  <slot>openColorWindow()</slot>
 </slots>
</ui>
